app/forms/state_file/id_disability_form.rb

```diff
@@ -3,11 +3,24 @@ class IdDisabilityForm < QuestionsForm
     set_attributes_for :intake, :primary_disabled, :spouse_disabled
 
     attr_accessor :mfj_disability
-    validates_presence_of :mfj_disability, if: -> { intake.filing_status_mfj?}
-    validates :primary_disabled, inclusion: { in: %w[yes no], message: :blank }, unless: -> { intake.filing_status_mfj? }
+    validates_presence_of :mfj_disability, if: -> { intake.filing_status_mfj? && intake.all_filers_between_62_and_65_years_old? }
+    validates :primary_disabled, inclusion: { in: %w[yes no], message: :blank }, if: -> { should_check_primary_disabled? }
+    validates :spouse_disabled, inclusion: { in: %w[yes no], message: :blank }, if: -> { should_check_spouse_disabled? }
 
-    def save
+    def should_check_primary_disabled?
       if intake.filing_status_mfj?
+        !intake.all_filers_between_62_and_65_years_old? && intake.primary_between_62_and_65_years_old?
+      else
+        intake.primary_between_62_and_65_years_old?
+      end
+    end
+
+    def should_check_spouse_disabled?
+      intake.filing_status_mfj? && !intake.all_filers_between_62_and_65_years_old? && intake.spouse_between_62_and_65_years_old?
+    end
```

Comment:
i _thought_ about moving these to the intake level, but feel like the names could be a bit misleading/confusing and the use case for these methods are really limited to this presence check.